parse_test.go

```diff
@@ -149,6 +149,25 @@ func TestParseFile_SkipProcessingPixelDataValue(t *testing.T) {
 			}
 		})
 	})
```

Comment:
I suppose it's fine to add this test case to this list, I really should generalize the tests some more. It may make sense to rename the "parent" test name which is currently `TestParseFile_SkipProcessingPixelDataValue` to be something like `TestParseFile_AcrossOptions` or something.

Reply:
Let me know what you decide but does it affect this PR?